synfig-studio/src/gui/widgets/widget_timeslider.cpp

```diff
@@ -303,23 +309,52 @@ Widget_Timeslider::on_draw(const Cairo::RefPtr<Cairo::Context> &cr)
 				double x1 = time_plot_data->get_double_pixel_t_coord(bounds[i][1]);
 				double w = x1 - x0;
 
+				double boundary_dimension=8;
+				double background_adjust;
+
+				if(i==0)
+					background_adjust=0;
+				else
+					background_adjust=8;
+
 				cr->save();
-				cr->rectangle(x0, 0.0, w, (double)get_height());
+				cr->rectangle(x0 + background_adjust, 0.0, w-8, (double)get_height());
```

Comment:
should this `8` and the following `8`'s be `boundary_dimension`?

Reply:
I believe so, I remember changing them all when changing the size.